extensions/wikia/AdEngine/js/Krux.run.js

```diff
@@ -2,33 +2,30 @@
 /*jshint camelcase:false*/
 require([
 	'ext.wikia.adEngine.adContext',
-	'ext.wikia.adEngine.krux',
+	'ext.wikia.krux',
+	'ext.wikia.adEngine.adLogicPageParams',
 	'jquery',
 	'wikia.log',
 	'wikia.scriptwriter',
 	'wikia.window'
-], function (adContext, Krux, $, log, scriptWriter, window) {
+], function (adContext, krux, adLogicPageParams, $, log, scriptWriter, window) {
```

Comment:
No more scriptWriter here please :-)

(In fact we shouldn't be needing this module at all, Krux.load should be called from the skins main run files.